### PR TITLE
NS-1422 Consolidate RDS profile indexing

### PR DESCRIPTION
**Pull request by pauline2k**
https://jira-secure.berkeley.edu/browse/NS-1422

### CODE REVIEW

File: nessie/sql_templates/update_rds_indexes_student_profiles.template.sql [hunk 1]
```diff
@@ -51,14 +74,32 @@ ON CONFLICT (sid) DO UPDATE SET
   sid=EXCLUDED.sid, uid=EXCLUDED.uid, first_name=EXCLUDED.first_name, last_name=EXCLUDED.last_name,
   level=EXCLUDED.level, gpa=EXCLUDED.gpa, units=EXCLUDED.units, transfer=EXCLUDED.transfer,
   expected_grad_term=EXCLUDED.expected_grad_term, terms_in_attendance=EXCLUDED.terms_in_attendance,
-  hist_enr=FALSE;
+  hist_enr=EXCLUDED.level;
```
Comment (lyttam) [inline nessie/sql_templates/update_rds_indexes_student_profiles.template.sql]: @pauline2k Does this work because hist_enr students don't have a level?

Comment (pauline2k) [reply nessie/sql_templates/update_rds_indexes_student_profiles.template.sql]: No, it doesn't work because I copied and pasted like a fool. One sec!

Comment (pauline2k) [reply nessie/sql_templates/update_rds_indexes_student_profiles.template.sql]: Amended.

Comment (lyttam) [reply nessie/sql_templates/update_rds_indexes_student_profiles.template.sql]: Ah! Thanks

**Pull request merged**